packages/data-point/lib/stores/entities.js

```diff
@@ -8,8 +8,8 @@ const storeManager = require('./store-manager')
 function createEntity (entityTypes, source, id) {
   const tokens = id.split(':')
   const entityType = tokens[0]
-  const EntityType = entityTypes.get(entityType)
-  const entity = EntityType.create(source, id)
+  const entityFactory = entityTypes.get(entityType)
```

Comment:
how about `const [entityType, entityName] = id.split(':')`? creating an `entityName` var would make it more clear what that value is

Reply:
smart! yes I like that